HOW_TO_CREATE_A_PLAN.md

```diff
@@ -0,0 +1,45 @@
+## How to create a plan - Contributor version
+
+### Preamble 
+If you’re new to working with Expensify, it’s important to know the philosophy behind the design of Expensify’s products and the processes that wrap around them.
+- Problems are identified and defined before solutions are proposed
+- Features are not added to satisfy personal preferences
+- Features are added thoughtfully and minimally
+
+It’s also important to be familiar with our existing documentation including our [CONTRIBUTING.md](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md) and [README.md](https://github.com/Expensify/App/blob/main/README.md)
+
+## Step 1: Define the problem
+What is the problem that needs solving? 
+- Be broad
+- Define who experiences the problem
+- Does this affect the customer experience? How so? 
+- How many customers would be affected by this?
+- Define why it is a problem
+- Resist the temptation to reverse-engineer a Solution into a matching Problem.  (ie. Problem: We don’t have a car. Solution: Buy a car) 
+- There may be more than one problem, so define all of them
+- Keep the language simple - Reduce every word down to its most plain and straightforward manner.  Avoid jargon and, if you use abbreviations, include the full spelling the first time.
+- Tear apart your statements and have others tear them apart for you until everybody agrees unambiguously as to the nature of the problem. If needed, continue the discussion in #expensify-open-source.
+
+Identify which problems are actually independent and can be solved by themselves, versus which are actually a series of dependent but otherwise separate problems that can be solved one at a time.
+
+The broader and better contextualized the problem definition, the greater opportunity for discovering radical, out-of-the-box solutions. The odds of achieving something are dramatically improved by clearly defining what it is.  Take at least as much care perfecting your problem definition as you do the solution, because the perfect solution to the wrong problem is typically an awful solution to the right one.
```

Comment:
I think there are two spaces here. Suggestion
```suggestion
The broader and better contextualized the problem definition, the greater opportunity for discovering radical, out-of-the-box solutions. The odds of achieving something are dramatically improved by clearly defining what it is. Take at least as much care perfecting your problem definition as you do the solution, because the perfect solution to the wrong problem is typically an awful solution to the right one.
```